single.php

```diff
@@ -34,13 +34,13 @@
 									endif;
 
 									if(get_the_content()):
-										echo "<p>".get_the_content()."</p>";
+											the_content();
```

Comment:
If the_content() is called without condition, it will add the meta tag and P tag as the screenshot below.
![screenshot_2](https://user-images.githubusercontent.com/6613530/28125051-a6cb6a38-674f-11e7-9706-69cdc145d560.jpg)